app/src/main/java/org/simple/clinic/storage/text/TextStoreModule.kt

```diff
@@ -2,24 +2,18 @@ package org.simple.clinic.storage.text
 
 import dagger.Module
 import dagger.Provides
-import org.simple.clinic.storage.files.FileStorage
-import javax.inject.Named
+import org.simple.clinic.AppDatabase
 
 @Module
 class TextStoreModule {
```

Comment:
Since `TextStore` is an interface and the `LocalDbTextStore` is an implementation, we will still need the module so that we can tell Dagger to instantiate `LocalDbTextStore` every time we ask for an instance of `TextStore`.